.test/data/.jsonoutput

```diff
@@ -0,0 +1,10 @@
+{
```

Comment:
I'll add to the docs how I update tests, but basically I just run `npm run start` to generate the summary file